src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -18,6 +25,11 @@
 
 public class ModerationCommands {
 
+  private static final int REPORT_COOLDOWN_SECONDS = 15;
+
+  private static final Cache<UUID, Instant> lastReportSent =
+      CacheBuilder.newBuilder().expireAfterWrite(REPORT_COOLDOWN_SECONDS, TimeUnit.SECONDS).build();
```

Comment:
I think this should be called `LAST_REPORT_SENT`.

Reply:
> I meant something like this: :P
> 
> https://github.com/Electroid/PGM/blob/31ebbcc74002981fcf05dc0af45e861a77d07aca/src/main/i18n/templates/strings.properties#L672-L674

Whoops! 😆 
I’ll make the adjustments now, thanks again haha.